docs/development/bpmn-support-how-to.md

```diff
@@ -107,7 +115,7 @@ Refer to existing Pull Requests to have a better view about the work to do, for
 
 All tests should have been introduced during the detection phase. Please review there is no missing tests.
 
-Visual tests introduced when adding the detection support should fail for the BPMN Element after the rendering has changed.
+Visual tests introduced when adding the detection support should fail for the BPMN Element after the rendering has changed. \
```

Comment:
```suggestion
Visual tests introduced when adding the detection support should fail for the BPMN Element after the rendering has changed.  
```

Reply:
I propose to use double space for the new line
like this, it is not visible in the raw .md code and still gives the desired effect on rendered .md

however it is subjective.
I would like that we decide with which approach we continue
@tbouffard can you give your opinion please

Reply:
Sometimes, you can remove the double space by mistake, and you don't see the effect.
I find the slash and clearer and avoid mistakes.

Reply:
Yes, I agree, this avoids unwanted automatic IDE formatting